packages/expo-yarn-workspaces/index.js

```diff
@@ -78,6 +79,61 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
   };
 };
 
+
+
+/**
+ * Returns a webpack configuration object that:
+ * 
+ *    * transpiles symlinked workspace packages
+ *    * watches for file changes in symlinked workspace packages
+ */
+exports.createWebpackConfigAsync = async function createWebpackConfigAsync(env, argv) {
```

Comment:
Maybe we should move this function to `expo-yarn-workspaces/webpack`, keeps the existing tooling more light-weight and the imports more organized.